modules/core/src/main/scala/org/scalasteward/core/application/Cli.scala

```diff
@@ -58,7 +58,8 @@ object Cli {
       githubAppKeyFile: Option[File] = None,
       githubAppId: Option[Long] = None,
       urlCheckerTestUrl: Option[Uri] = None,
-      defaultMavenRepo: Option[String] = None
+      defaultMavenRepo: Option[String] = None,
+      refreshBackoffPeriod: Option[FiniteDuration] = None
```

Comment:
I wasn't sure if set the default at this level or when creating the `Config`. Might make more sense to set it here than where I wrote it

Reply:
Most default values are set here, so I guess it makes sense to also set it here for `refreshBackoffPeriod`.

Reply:
Yeah, I will change that, thanks!